packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -351,6 +352,32 @@ export async function getBookings({
       take: take + 1,
       skip,
     }),
+    prisma.booking.findMany({
+      where: {
+        OR: [
```

Comment:
why do we need OR here? there is only one query inside OR array

Reply:
I don't think we do.